hadoop-hdds/common/src/main/java/org/apache/hadoop/utils/db/DBStoreBuilder.java

```diff
@@ -187,7 +196,13 @@ private DBOptions getDbProfile() {
 
     if (option == null) {
       LOG.info("Using default options. {}", dbProfile.toString());
-      return dbProfile.getDBOptions();
+      option = dbProfile.getDBOptions();
+    }
+
+    if (!rocksDbStat.equals(OZONE_METADATA_STORE_ROCKSDB_STATISTICS_OFF)) {
```

Comment:
There is some history here. During our first release we found that RocksDB is also shipped by YARN. That version of RocksDB is very old, hence this call would fail in mysterious ways. @arp7  went and fixed that issue and made sure that we don't enable this by default. I am fine with enabling this, if we don't run into that old issue again. @elek, @arp7 any comments ?

Reply:
@anuengineer We don't enable the metrics by default. I just used the same config that is used to enable metrics for SCM RocksDB, to enable metrics for OM RocksDB as well.

Reply:
I am not familiar with the old issue, but now our classpath is separated from yarn. We can use any newer rocksdb (but we should use the same versions for all the hdds/ozone projects)

I think it's safe to use any new rocksdb feature if you think metrics should be turned on by default...